### ISSUE TITLE
Docs: reintroduce the different missing parts from Bootstrap

### ISSUE THREAD

**Issue by louismaximepiton** [Orange-OpenSource/Orange-Boosted-Bootstrap#1856]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

After #1843,

Reintroduce the following inside one PR for each:
- [ ] Functional backgrounds with the new colors based on light/dark theme.
- [ ] Floating labels (better for theming)
- [ ] Flush accordions that might be useful sometimes.
- [ ] Others ? Need to check deeper

### Motivation and context

Better theming for the different themes we might have.

**Comment by MaxLardenois** [comment]
Boosted, won't fix
